packages/devtools_app/lib/src/timeline/timeline_model.dart

```diff
@@ -997,7 +997,7 @@ class AsyncTimelineEvent extends TimelineEvent {
         math.min(event.displayDepth, currentLargestRowIndex - displayRow);
     for (int level = 0; level < maxLevelToVerify; level++) {
       final lastEventAtLevel = _displayRows[displayRow + level].safeLast;
-      final firstNewEventAtLevel = event.firstChildNodeAtLevel(level);
+      final firstNewEventAtLevel = event.displayRows[level].first;
```

Comment:
shouldn't this be .safeFirst to avoid a NPE if the row is empty?
If the now is never entry then the `firstNewEventAtLevel != null` check should probably be removed.

Reply:
Used safeFirst